# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1406215399 -3600
# Node ID 784e7e8a5f76fff96df678a5620e6bb84644155b
# Parent  ad4c824a25ce205fe98f89320f751e1971fc906a
tools/xenconsoled: Newline on 'Logfile Opened' messages

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
CC: Ian Campbell <Ian.Campbell@citrix.com>
CC: Ian Jackson <Ian.Jackson@eu.citrix.com>
Acked-by: Ian Campbell <ian.campbell@citrix.com>

diff -r ad4c824a25ce -r 784e7e8a5f76 tools/console/daemon/io.c
--- a/tools/console/daemon/io.c
+++ b/tools/console/daemon/io.c
@@ -276,8 +276,8 @@ static int create_hv_log(void)
 		dolog(LOG_ERR, "Failed to open log %s: %d (%s)",
 		      logfile, errno, strerror(errno));
 	if (fd != -1 && log_time_hv) {
-		if (write_with_timestamp(fd, "Logfile Opened",
-					 strlen("Logfile Opened"),
+		if (write_with_timestamp(fd, "Logfile Opened\n",
+					 strlen("Logfile Opened\n"),
 					 &log_time_hv_needts) < 0) {
 			dolog(LOG_ERR, "Failed to log opening timestamp "
 				       "in %s: %d (%s)", logfile, errno,
@@ -322,8 +322,8 @@ static int create_domain_log(struct doma
 		dolog(LOG_ERR, "Failed to open log %s: %d (%s)",
 		      logfile, errno, strerror(errno));
 	if (fd != -1 && log_time_guest) {
-		if (write_with_timestamp(fd, "Logfile Opened",
-					 strlen("Logfile Opened"),
+		if (write_with_timestamp(fd, "Logfile Opened\n",
+					 strlen("Logfile Opened\n"),
 					 &log_time_guest_needts) < 0) {
 			dolog(LOG_ERR, "Failed to log opening timestamp "
 				       "in %s: %d (%s)", logfile, errno,
